src/react-apps/applications/runtime/src/features/instantiate/containers/PartySelection.tsx

```diff
@@ -182,12 +181,12 @@ function PartySelection(props: IPartySelectionProps) {
     if (!language || !language.party_selection) {
       return null;
     }
-    if (location.state !== undefined &&
-      (location.state as IRedirectReason) &&
-      (location.state as IRedirectReason) !== undefined) {
-        switch ((location.state as IRedirectReason).errorType) {
+    if ((match.params as IRedirectParams).errorCode !== null) {
+      try {
```

Comment:
Hva med å fjerne try catch og heller sjekke på isNaN på errorCode? Er det mulig mtp TypeScript når man har definert en const som number? Kan den bli satt til NaN? Eller vil det da krasje? Og det er derfor du har try catchen?

Reply:
In Typescript this will make an error:
```
const number: number = '1';
```
**BUT**  we're not running typescript in the browser. TypeScript is transpiled to JavaScript when we build. So in runtime that will work.